workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/features/project/pages/Project/routes/project.routes.jsx

```diff
@@ -79,13 +79,21 @@ const ProjectRoutes = () => {
 	const loggedUserAccount = myUserAccountData?.myUserAccount;
 
 	const hasSaasSubscription = useMemo(
-		() =>
-			subscriptionGroups?.some(
-				(subscription) =>
-					subscription.externalReferenceCode ===
-					`${project?.externalReferenceCode}_liferay-saas`
-			),
-		[subscriptionGroups]
+		() =>  {
+			const legacySaasERC = `${project?.externalReferenceCode}_liferay-saas`;
+			const consolidatedCloudERC = `${project?.externalReferenceCode}_liferay-cloud`;
+
+			return subscriptionGroups?.some(
+				(subscription) => {
+					const erc = subscription.externalReferenceCode;
```

Comment:
Wouldn't something less generic be better, following the pattern of the variables above?